war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1177,7 +1177,7 @@ function rowvgStartEachRow(recursive,f) {
 
     // Native browser resizing doesn't work for CodeMirror textboxes so let's create our own
     Behaviour.specify(".CodeMirror", "codemirror", ++p, function(codemirror) {
-      const MIN_HEIGHT = 200;
+      const MIN_HEIGHT = codemirror.clientHeight;
```

Comment:
This can be a problem for input fields like Execute Shell, whose default size changes based on the size of the script content. Maybe some folks would like to make this smaller temporarily, but can't:

> <img width="1575" alt="Screenshot 2022-08-16 at 12 44 32" src="https://user-images.githubusercontent.com/1831569/184861195-27a2877c-4fed-4654-bd0b-8c6b7f8fd20b.png">

Probably

```suggestion
      const MIN_HEIGHT = Math.min(200, codemirror.clientHeight);
```

or so?

Reply:
Yess. With this, user can reduce the height of fields to 200px. Thanks!